test/specs/kefir-observable.js

```diff
@@ -0,0 +1,49 @@
+let {Kefir} = require('../test-helpers.coffee');
+
+describe('Kefir.Observable', function() {
+    describe('observe', function() {
+        let em, count, obs, sub;
```

Comment:
We want to be able to run them in browsers. We have that link in readme: 
![image](https://cloud.githubusercontent.com/assets/825702/16616212/75a59176-4384-11e6-8c9d-25f2b7abdc0b.png)

Ability to run test in older node versions is also nice, in case we'll want to check if we still support them.